From 5ae5f45ce26e9284f3619c1ee66de05d407fda0c Mon Sep 17 00:00:00 2001
From: Eddie Huang <eddie.huang@mediatek.com>
Date: Mon, 2 Feb 2015 19:46:22 +0800
Subject: [PATCH 8/9] Fix: arm64: dts: mt8173: Add pinctrl/GPIO/EINT node for
 mt8173.

Add pinctrl,GPIO and EINT node to mt8173.dtsi.
- Add pins-are-numbered

Change-Id: Idd762617f1302805e5ea59b507056268b21408bb
Signed-off-by: Eddie Huang <eddie.huang@mediatek.com>
---
 arch/arm64/boot/dts/mediatek/mt8173.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/mediatek/mt8173.dtsi b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
index 3d195c4..f2114f9 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
@@ -115,6 +115,7 @@
 			compatible = "mediatek,mt8173-pinctrl";
 			reg = <0 0x1000B000 0 0x1000>;
 			mediatek,pctl-regmap = <&syscfg_pctl_a>;
+			pins-are-numbered;
 			gpio-controller;
 			#gpio-cells = <2>;
 			interrupt-controller;
-- 
1.9.1

